pyrsistent/_pclass.py

```diff
@@ -12,7 +12,7 @@ class PClassMeta(type):
     def __new__(mcs, name, bases, dct):
         set_fields(dct, bases, name='_pclass_fields')
         store_invariants(dct, bases, '_pclass_invariants', '__invariant__')
-        dct['__slots__'] = ('_pclass_frozen',) + tuple(key for key in dct['_pclass_fields'])
+        dct['__slots__'] = ('_pclass_frozen','_hash',) + tuple(key for key in dct['_pclass_fields'])
```

Comment:
We should probably use a longer name than just '_hash'.  Something like '_pclass_hash' or something.

Otherwise I'm not sure how this would interact with a PClass subclass that has a field named '_hash'.